pkg/auth/always_pass_authhandler.go

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package auth
+
+import (
+	"time"
+
+	"github.com/cilium/cilium/pkg/auth/certs"
+	"github.com/cilium/cilium/pkg/policy"
+)
+
+// alwaysPassAuthHandler implements an authHandler for auth type disable by just authenticate every request.
```

Comment:
nit: comment is wrong regarding auth type `disable` -> maybe document that this one is for testing and lets every auth request pass without any check.